recipes/_install-windows.rb

```diff
@@ -24,17 +24,19 @@
 temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
 
 # Download the installer to a temp location
-remote_file 'MSI installer' do
-  path temp_file
+remote_file temp_file do
   source node['datadog']['windows_agent_url'] + dd_agent_msi
+  checksum node['datadog']['windows_agent_checksum'] if node['datadog']['windows_agent_checksum']
```

Comment:
Thanks for the detailed explanation! Actually you're right, the resource _does_ check that the downloaded file matches the provided checksum, and fails if not (although the docs only mention that the checksum is used to decide whether to re-download the existing file or not).

Could you add a comment on how to use the attribute in `attributes/default.rb`?